docs/example-select.js

```diff
@@ -0,0 +1,19 @@
+function switchTab(event) {
```

Comment:
This is highly coupled with this use case. I guess it's fine if this isn't supposed to be reused.

Reply:
I suppose i could actually name this stuff as "tabs" and have it be super generic?